packages/turbo-repository/rust/src/lib.rs

```diff
@@ -63,4 +98,21 @@ impl Workspace {
     pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
         self.packages_internal().await.map_err(|e| e.into())
     }
+
+    #[napi]
+    pub async fn package_graph(&self) -> Result<HashMap<String, Vec<String>>, Error> {
+        let mut map = HashMap::new();
+        let packages = self.find_packages().await?;
+        let workspace_path = AbsoluteSystemPath::new(self.absolute_path.as_str()).unwrap();
+
+        for (_i, package) in packages.iter().enumerate() {
```

Comment:
Not required - but you could map this as well, and then return tuples that get collected into key val pairs to avoid the mutable hashmap

```
let map: HashMap<String, Vec<String>> = packages.iter()
        .map(|package| {
            let deps = package.dependents(&self.graph, workspace_path);
            let dep_names = deps.iter().map(|p| p.relative_path.clone()).collect::<Vec<String>>();

            (package.relative_path.clone(), dep_names)
        })
        .collect();
```

Reply:
good idea, done